Homework 1/DuodecimalFinder.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace Homework_1
+{
```

Comment:
Please redo the task not using standard collections and LINQ, only array are allowed. And simplify, too many unclear unnecessary methods, the task is more simple then you think ))